src/shared/components/simple-select/SimpleSelect.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/styles/common';
+
```

Comment:
Is there anything in this component that we want to make tweakable via custom properties?

Reply:
I've made the following properties tweakable:
- min-width: var(--select-min-width, 15ch);
- max-width: var(--select-max-width, 30ch);
- background-color: var(--select-background-color, $white);